src/styles/blocks/stars.css

```diff
@@ -0,0 +1,39 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("../../images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
the last star shouldn't have this margin, please add only for elements, which are not last of type :wink: